packages/client/modules/teamDashboard/containers/Team/TeamContainer.tsx

```diff
@@ -61,7 +61,7 @@ const TeamContainer = (props: Props) => {
             <Route
               path={`${match.path}/archive`}
               // render={(p) => <ArchivedTasks {...p} team={team} />}
-              render={(p) => <ArchivedTasks {...p.match.params} team={team} showHeader={true} />}
+              render={(p) => <ArchivedTasks {...p} team={team} returnToTeamId={teamId} teamIds={[teamId]} />}
```

Comment:
whenever using the renderProps pattern with react router we've gotta pass through all the props, otherwise the router won't always update correctly (so always {...p})